pom.xml

```diff
@@ -170,6 +170,11 @@
             <scope>test</scope>
             <version>3.0.3</version>
         </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>msal4j</artifactId>
+            <version>0.5.0-preview</version>
```

Comment:
This is very old version. Let's use the latest one.

Reply:
changed to the latest 1.7.0